commit d4619b77a669c70ae68d5da34de7655ffef80e04
Author: Michael Prokop <mika@grml.org>
Date:   Mon Jul 11 12:42:24 2011 +0200

    Try loading phram module twice to fix loading issue.
    
    phram doesn't work on first modprobe, when loading once again it seems
    to work. So let's try it this way.

--- a/scripts/boot/9990-main.sh
+++ b/scripts/boot/9990-main.sh
@@ -1576,6 +1576,7 @@
 				then
 					# We found a memdisk, set up phram
 					modprobe phram phram=memdisk,${MEMDISK}
+					modprobe phram phram=memdisk,${MEMDISK}
 
 					# Load mtdblock, the memdisk will be /dev/mtdblock0
 					modprobe mtdblock
